src/components/AdminDashboard/AntModalComponent/AntModal.js

```diff
@@ -1,55 +1,52 @@
-import React, { useState } from 'react';
-import { Modal, Button, Input } from 'antd';
+import React, { useEffect, useState } from 'react';
+import { Modal, Button, Input, Checkbox } from 'antd';
+import './AntModal.css';
 import axios from 'axios';
 
 // This component returns an antd modal that has an antd input component nested within it. The input's text area is prepopulated with a default message that can be edited. When the send button is clicked, the message is sent to the Twitter Bot on the DS Backend.
-const AntModal = () => {
-  // The default message displayed inside of the input's text area
-  const defaultMessage = {
-    message: 'Hello user!',
-  };
+const AntModal = props => {
+  const incident_id = props.incident.incident_id;
+  const tweet_id = props.incident.tweet_id;
+  const user_name = props.incident.user_name;
 
   const [isModalVisible, setIsModalVisible] = useState(false);
-  const [inputValue, setInputValue] = useState(defaultMessage);
   const [isButtonDisabled, setIsButtonDisabled] = useState(false);
 
+  const sendObj = {
+    incident_id: incident_id,
+    tweet_id: tweet_id,
+    isChecked: true,
+    user_name: user_name,
+    link: `https://a.humanrightsfirst.dev/edit/${tweet_id}`,
+    form: 1,
+  };
+
   const showModal = () => {
     setIsModalVisible(true);
   };
 
   const handleSend = () => {
-    setIsModalVisible(false);
+    console.log(sendObj);
 
-    axios
-      .post(
-        'https://run.mocky.io/v3/f1175471-f517-4935-9d53-5319d0c95b3d',
-        inputValue
-      )
-      .then(res => {
-        console.log(res);
-      })
-      .catch(err => {
-        console.log(err);
-      });
-  };
+    // axios
+    //   .post(
+    //     'http://hrf-bw-labs37-dev.eba-hz3uh94j.us-east-1.elasticbeanstalk.com/form-out',
+    //     sendObj
+    //   )
+    //   .then(res => {
+    //     console.log(res);
+    //   })
+    //   .catch(err => {
+    //     console.log(err);
+    //   });
```

Comment:
Everything looks awesome! Nice work guys! I was wondering about this section though. Is the commented out part for the next Labs group? Also there is a console.log on line 29